tests/db/schemas/mysql.sql

```diff
@@ -133,7 +133,7 @@ CREATE TABLE trace_info (
 	timestamp_ms BIGINT NOT NULL,
 	execution_time_ms BIGINT,
 	status VARCHAR(50) NOT NULL,
-	CONSTRAINT trace_info_pk PRIMARY KEY (request_id),
+	PRIMARY KEY (request_id),
```

Comment:
@harupy can you double check if this is okay? Alembic is automatically removing the primary key constraint name in the mysql schema for the trace tables despite the migration script in this PR not touching the trace tables at all. It looks like mysql doesn't allow for primary key constraints to be named?